miniapp/src/main/res/values-ja/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="miniapp.sdk.android.locale">ja-JP</string>
```

Comment:
I think this should use `_` to match the other strings.

Reply:
Thanks, I have updated it 🙇